PP_APP/server/frontend/static/js/script.js

```diff
@@ -3,126 +3,87 @@ let user_email = document.getElementById('user_email');
-                $("#res_img").attr("src", data.response.infer_img)
-                $("#post_infer").css("display", "block")
-                $("#score").text('6.5/10')
-                $("#outer_size").text(data.response.outer_size)
-                $("#inner_size").text(data.response.inner_size)
-                $("#item_size").text(data.response.item_size)
-                let elapsed = Math.floor(((new Date().getTime() - now) % (1000 * 60)) / 1000)
-                $("#time_elapsed").text(elapsed)
-                email_val = user_email.value
-                // store EMAIL locally so the user doesn't to retype in the email on refresh
-                localStorage.setItem("user_email", email_val);
-                document.getElementById("imageUploadForm").reset();
-                user_email.setAttribute('value', email_val);
-
-
-                $('#top-dropzone').attr('style', 'background:linear-gradient(to bottom, rgba(22, 22, 22, 0.5) 0%, rgba(22, 22, 22, 0.8) 80%, #0000008a 100%), url(/static/img/sample_top.jpg) top left no-repeat;background-size:cover;');
-                $('#side-dropzone').attr('style', 'background:linear-gradient(to bottom, rgba(22, 22, 22, 0.5) 0%, rgba(22, 22, 22, 0.8) 80%, #0000008a 100%), url(/static/img/sample_side.jpg) top left no-repeat;background-size:cover;');
-
-            },
-            error: function (e) {
-              $("#status").text(JSON.parse(e.responseText).response);
-                // $("#status").text('Please try again.');
-                $("#btnSubmit").prop("disabled", false);
-                $("#res_img").attr("src", "")
-                email_val = user_email.value
-                localStorage.setItem("user_email", email_val);
-                document.getElementById("imageUploadForm").reset();
-                user_email.setAttribute('value', email_val)
-                $('#top-dropzone').attr('style', 'background:linear-gradient(to bottom, rgba(22, 22, 22, 0.5) 0%, rgba(22, 22, 22, 0.8) 80%, #0000008a 100%), url(/static/img/sample_top.jpg) top left no-repeat;background-size:cover;');
-                $('#side-dropzone').attr('style', 'background:linear-gradient(to bottom, rgba(22, 22, 22, 0.5) 0%, rgba(22, 22, 22, 0.8) 80%, #0000008a 100%), url(/static/img/sample_side.jpg) top left no-repeat;background-size:cover;');
-            }
-        });
+      //stop submit the form, we will post it manually.
+      event.preventDefault();
+
+      // Create an FormData object 
+      let mrcnn_in_data = new FormData();
+      let gv_in_data_top = new FormData();
+      let gv_in_data_side = new FormData();
 
+      // If you want to add an extra field for the FormData
+      mrcnn_in_data.append("packager", "DummyBrand");
+      mrcnn_in_data.append("email", user_email.value);
+      mrcnn_in_data.append("top_img", top_input.files[0]);
+      mrcnn_in_data.append("side_img", side_input.files[0]);
+      
+      gv_in_data_top.append("side_view", side_input.files[0]);
```

Comment:
This should be: 
`gv_in_data_top.append("side_view", top_input.files[0]);`
if you want to enter the top image.

I apologize for the misnomer here; the side_view field is not the best naming practice.

Reply:
Got it. Fixed.